src/index.test.ts

```diff
@@ -0,0 +1,7 @@
+import * as sdk from '.';
+
+describe('SDK', () => {
+  it('exports payments', () => {
+    expect(sdk).toHaveProperty('payments');
```

Comment:
this test verifies we're exposing what (and how) we think but also that types build correctly